components/events/EventCard.vue

```diff
@@ -65,96 +65,108 @@ export default class extends Vue {
 <style lang="scss" scoped>
 @import '~carbon-components/scss/globals/scss/typography';
 
-.card-link {
-  text-decoration: none;
-}
+$img-medium-width: 13rem;
 
 .event-card {
-  height: 15.88rem;
+  min-height: 13rem;
   width: 100%;
-  margin-bottom: 1rem;
-  background-color: $ui-01;
-  color: $text-01;
-  border-top: 1px solid $ui-03;
+  margin-bottom: $layout-03;
+  background-color: $cool-gray-10;
+  color: $cool-gray-80;
   display: flex;
 
   @include mq($until: medium) {
     height: auto;
-    flex-direction: column-reverse;
-  }
-
-  &__content {
-    flex: 1;
-    margin: 1rem;
-    display: flex;
     flex-direction: column;
-    justify-content: space-between;
   }
 
-  &__title {
-    @include type-style('productive-heading-03');
-    margin-top: 0.31rem;
-  }
-
-  &__subtitle {
-    @include type-style('productive-heading-02');
-  }
-
-  &__location, &__date {
-    @include type-style('body-short-01');
-  }
+  &__image {
+    flex: 0 0 14rem;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    overflow: hidden;
 
-  &__location, &__date {
-    padding-left: 0.31rem;
-  }
+    @include mq($from: medium, $until: large) {
+      flex: 0 0 13rem;
+    }
 
-  &__info-detail {
-    margin-top: 0.63rem;
+    @include mq($until: medium) {
+      height: 13rem;
+      width: auto;
+    }
   }
 
-  &__icon {
-    fill: currentColor;
-    position: relative;
-    top: .3em;
-    width: 1.25rem;
-    height: 1.25rem;
+  &__content {
+    @include type-style('body-long-01');
+    padding: $spacing-05 $spacing-05 $spacing-05 $spacing-07;
 
-    &--purple {
-      color: $icon-01;
+    @include mq($from: medium, $until: large) {
+      width: calc(100% - #{$img-medium-width});
```

Comment:
I'm good with this since it's derived from a calculation that we can understand...